### PR TITLE
add 4-basics

### PR DESCRIPTION
**Pull request by DavidTereshchenko**
I did it this way because I didn't have time
tomorrow I will try to do it from DOM

### CODE REVIEW

File: homeworks/david.tereshchenko_DavidTereshchenko/4-basics/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>JS</title>
+    </head>
+    <body>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/4-basics/index.html:7]: Use HTML for rendering the results. HTML loads faster, allowing users to see some content while the page is loading. Please create the HTML content in the index.html file

Comment (anna-vrana-moc) [reply homeworks/david.tereshchenko_DavidTereshchenko/4-basics/index.html:7]: Get HTML element by data-attributes

File: homeworks/david.tereshchenko_DavidTereshchenko/4-basics/basics.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const firstNumber = +prompt('Enter the first number', '').trim();
+const operator = prompt('Enter mathematics operator (+, -, *, /, %, **,)', '').trim();
+const secondNumber = +prompt('Enter the second number', '').trim();
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/4-basics/basics.js:3]: It is so many prompts. Use it to get the numbers and render all results on the page

**Review by anna-vrana-moc — CHANGES_REQUESTED** [review]

**Comment by anna-vrana-moc** [comment]
Please fix linter errors

**Review by anna-vrana-moc — APPROVED** [review]

**Comment by anna-vrana-moc** [comment]
Hey @DavidTereshchenko great job! I like it) For the future get HTML elements by data-attributes, like data-result, because somebody can remove your class to fix css and broke the logic

**Comment by DavidTereshchenko** [comment]
ok, I'll pay attention to that, thanks

**Pull request merged**